fontforge/sfd.c

```diff
@@ -5763,9 +5773,14 @@ return( NULL );
 			lasti->next = img;
 		    lasti = img;
 		}
-	    } else
+	    } else {
```

Comment:
You can simplify/remove the need for that other ifdef by moving this bracket below the `#endif` and unconditionally including the closing `#endif`, ditto for the other location

Reply:
Wow, that's cool! I didn't know `{ stmt; stmt; stmt; }` was valid without a preceding keyword; thought only Rust did that for some reason!